axum-macros/tests/from_request/pass/named_via.rs

```diff
@@ -25,7 +25,7 @@ struct Extractor {
 
 fn assert_from_request()
 where
-    Extractor: FromRequest<Body, Rejection = ExtractorRejection>,
+    Extractor: FromRequest<Body, (), Rejection = ExtractorRejection>,
```

Comment:
Just noticed this: I think the order of `B` and `S` in `FromRequest` and `Router` should match.

Reply:
Good point! I agree. The order on `FromRequest` and `RequestParts` was a hold over from when we used to have `S = ()`. So that had to be last since we don't have a default for `B`.

I don't think we should have a default type param for `FromRequest` and `RequestParts` because I'd worry library authors accidentally doing

```rust
impl<B> FromRequest<B> for Foo {}
```

without realizing that their extractor now only works for `Router<()>`.

I'll fix it (and change all the files again 🥲)

Reply:
Done!